core/src/test/java/org/apache/iceberg/TestMetadataUpdateParser.java

```diff
@@ -52,6 +56,15 @@ public class TestMetadataUpdateParser {
           Types.NestedField.required(1, "id", Types.IntegerType.get()),
           Types.NestedField.optional(2, "data", Types.StringType.get()));
 
+  private static final Schema ID_VARIANTDATA_SCHEMA =
```

Comment:
This isn't the right place for a Variant test. What are you trying to test?

Reply:
I'm trying to test SchemaParser indirectly. I don't see TestSchemaParser.